inflammation/models.py

```diff
@@ -32,3 +32,13 @@ def daily_min(data):
     """Calculate the daily min of a 2d inflammation data array."""
     return np.min(data, axis=0)
 
+
+def s_dev(data):
```

Comment:
The function name is not very descriptive. Maybe something like daily_stddev is more descriptive and in line with the other functions in model.py

Reply:
Resolved with commit bcc8a02